lib/tests/streamlit/elements/data_editor_test.py

```diff
@@ -565,14 +565,12 @@ def test_pandas_styler_support(self):
             data=np.arange(0, 6, 1).reshape(2, 3),
         )
         styler = df.style
-        # NOTE: If UUID is not set - a random UUID will be generated.
-        styler.set_uuid("FAKE_UUID")
         styler.highlight_max(axis=None)
-        st.data_editor(styler)
+        st.data_editor(styler, key="styler_editor")
 
         proto = self.get_delta_from_queue().new_element.arrow_data_frame
         self.assertEqual(
-            proto.styler.styles, "#T_FAKE_UUIDrow1_col2 { background-color: yellow }"
+            proto.styler.styles, "#T_29028row1_col2 { background-color: yellow }"
```

Comment:
Would it be worth adding a test that asserts that:
- The instances with the same delta path hash to the same UUID
- Two instances with different paths hash to different UUIDs
?

Reply:
Added some additional tests 👍